src/content/changes/highlights/2024-10-10.md

```diff
@@ -0,0 +1,67 @@
+
+---
+date: 2024-10-10T14:00:00
+title: Highlights for the week ending 2024-10-10
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Observability
```

Comment:
@Rotfuks @QuentinBisson  if you want to double-check or suggest any improvement on the messages please do